lib/src/importer/async.dart

```diff
@@ -35,9 +35,21 @@ abstract class AsyncImporter {
   /// * If the [url] ends in `.sass` or `.scss`, the importer should look for
   ///   a stylesheet with that exact URL and return `null` if it's not found.
   ///
-  /// * Otherwise, the importer should look for a stylesheet at `"$url.sass"` or
-  ///   one at `"$url.scss"`, in that order. If neither is found, it should
-  ///   return `null`.
+  /// * Otherwise, the importer should look for a stylesheet by filling in
+  ///   extensions and partial prefixes. The extension `.sass` before `.scss`,
+  ///   and using the partial prefix before without the prefix:
+  ///   * `"_$url.sass"`
+  ///   * `"$url.sass"`
+  ///   * `"_$url.scss"`
+  ///   * `"$url.scss"`
+  ///
+  /// * Finally, the importer should check for a directory default index.
+  ///   * `"$url<File System Delimiter>_index.sass"`
+  ///   * `"$url<File System Delimiter>index.sass"`
+  ///   * `"$url<File System Delimiter>_index.scss"`
+  ///   * `"$url<File System Delimiter>index.scss"`
```

Comment:
I think we should use `/` rather than `<File System Delimiter>` here, since we're dealing with URLs rather than paths. As a bonus, it's also easier to read!

Reply:
Sounds good. Done.